decidim-budgets/config/locales/en.yml

```diff
@@ -0,0 +1,102 @@
+---
+en:
+  decidim:
+    budgets:
+      actions:
+        confirm_destroy: Are you sure you want to delete this project?
+        destroy: Delete
+        edit: Edit
+        new: New %{name}
+        title: Actions
+      admin:
+        models:
+          project:
+            name: Project
+        projects:
+          create:
+            invalid: There's been a problem creating this project
+            success: Project successfully created
+          destroy:
+            success: Project successfully created
+          edit:
+            title: Edit project
+            update: Update project
+          index:
+            title: Projects
+          new:
+            create: Create project
+            title: New result
+          update:
+            invalid: There's been a problem updating this project
+            success: Project successfully updated
+      models:
+        project:
+          fields:
+            title: Title
+      projects:
+        budget_confirm:
+          are_you_sure: "¿Estás de acuerdo? Una vez hayas confirmado tu voto, no se
```

Comment:
We are waiting for the final texts but I think @josepjaume wanted to translate it by himself.